### PR TITLE
minus method added

### PR DESCRIPTION
**Pull request by axrortatu**

### CODE REVIEW

File: git_advance/src/Main.java [hunk 1]
```diff
@@ -2,9 +2,14 @@ public class Main {
     public static void main(String[] args) {
         System.out.println("Hello world!");
         sum(3,4);
+        minussss(10,4);
```
Comment (Jamoliddin007) [inline git_advance/src/Main.java]: please correct the method name

File: git_advance/src/Main.java [hunk 2]
```diff
@@ -2,9 +2,14 @@ public class Main {
...
     }
 
     public static void sum(int a, int b) {
         System.out.println(a + b);
     }
+
+    public static void minussss(int a, int b) {
```
Comment (Jamoliddin007) [inline git_advance/src/Main.java]: please correct the method name

File: git_advance/src/Main.java [hunk 3]
```diff
@@ -2,9 +2,14 @@ public class Main {
...
+        System.out.println(a + b);
```
Comment (Jamoliddin007) [inline git_advance/src/Main.java]: it should be `a-b`

File: git_advance/src/Main.java (hunk 1 above)
Comment (axrortatu) [reply git_advance/src/Main.java]: fixed

File: git_advance/src/Main.java (hunk 2 above)
Comment (axrortatu) [reply git_advance/src/Main.java]: fixed

File: git_advance/src/Main.java (hunk 3 above)
Comment (axrortatu) [reply git_advance/src/Main.java]: fixed

**Review by Jamoliddin007 — APPROVED** [review]

**Pull request merged**